scm/driver/bitbucket/webhook.go

```diff
@@ -598,6 +607,137 @@ type (
 		Repository  prCommentHookRepo        `json:"repository"`
 		Actor       prCommentHookUser        `json:"actor"`
 	}
+
+	pipelineHook struct {
+		Repository struct {
+			Type     string `json:"type"`
+			FullName string `json:"full_name"`
+			Links    struct {
+				Self   href `json:"self"`
+				HTML   href `json:"html"`
+				Avatar href `json:"avatar"`
+			} `json:"links"`
+			Name      string    `json:"name"`
+			SCM       string    `json:"scm"`
+			Website   *string   `json:"website"`
```

Comment:
remove pointer, use `null.String` instead (see examples in other files)

Reply:
Updated the type